.isort.cfg

```diff
@@ -1,3 +0,0 @@
-[settings]
-profile = black
-line_length = 79
```

Comment:
What happened to the line length limit in the isort configuration?

Reply:
Pretty sure isn't needed because `profile=black` sets it to the right one

Reply:
> Pretty sure isn't needed because `profile=black` sets it to the right one

Black uses 88 by default

Reply:
It copies the line length for black. I just checked and isort complains about a length 83 line with the config i set up.

Reply:
Oh nice, so it actually looks for the black config.